amqpdispatcher/message.py

```diff
@@ -73,5 +73,5 @@ def properties(self):
     def __str__(self):
         return ("Message[body: %s, delivery_info: %s, return_info: %s, "
                 "properties: %s]") %\
-            (str(self._body).encode('string_escape'),
+            (str(self._body).encode('utf-8'),
```

Comment:
For backwards compatibility, can you wrap this in a check for python 2 vs python3? Basically use `six.PY2` from the `six` package and do the old thing if possible.